charts/spire/charts/spire-server/templates/roles.yaml

```diff
@@ -1,15 +1,13 @@
-# ClusterRole to allow spire-server node attestor to query Token Review API
-# and to be able to push certificate bundles to a configmap
+# ClusterRole to allow spire-server node attestor to query Token Review API,
+# Role to be able to push certificate bundles to a configmap, and Role to
+# be able to manage cert requests with Cert-Manager
+---
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ include "spire-server.fullname" . }}
-  namespace: {{ .Release.Namespace }}
+  name: {{ include "spire-server.fullname" . }}-bundle
+  namespace: {{ .Values.notifier.k8sbundle.namespace | default .Release.Namespace }}
 rules:
-  # allow "get" access to pods (to resolve selectors for PSAT attestation)
-  - apiGroups: [""]
-    resources: [pods]
-    verbs: [get]
```

Comment:
The role code here is removed, cause it never worked. It is a Role, so only applies to the spire-server's namespace. Its the cluster role below that lets the pods get PSAT'ed for the entire cluster. That remains and is functional.

Reply:
Nit pick, can we move this into a separate PR. For the sake of traceability and release notes.

Our CONTRIBUTING.md also states that PRs should have a single subject. This change does not align with that PR title.

Reply:
sigh

nit picking the nit pick a little bit. Its going to take a lot more effort to add another role just to remove it again.We're early in the project and we're slowing things way down with nitpicking the really small stuff.

I can kind of live with this though.

But, one of the things I'm really objecting to is the little feature patches having to merge before the tests that ensure they work. That is BAD imo. I broke this off a working pr with tests ensuring it works. now there are changes being asked for and there is no tests to ensure the changes work. So, I think the current process needs fixing.

Reply:
https://github.com/spiffe/helm-charts/pull/104

Reply:
done.